1_fetch/src/fetch_river_geometry.R

```diff
@@ -1,38 +1,34 @@
-fetch_major_river_geoms <- function(ind_file, view_polygon, pre_simplify_m) {
+fetch_major_river_geoms <- function(ind_file, view_polygon, fetch_streamorder) {
 
   bbox <- st_bbox(st_transform(view_polygon, 4326))
 
   postURL <- "https://cida.usgs.gov/nwc/geoserver/nhdplus/ows"
 
-  # streamorder <- 4
-  # Leaving this here in case we want to add a property filter back
-  # '<ogc:And>',
-  # '<ogc:PropertyIsGreaterThan>',
-  # '<ogc:PropertyName>streamorde</ogc:PropertyName>',
-  # '<ogc:Literal>',streamorder,'</ogc:Literal>',
-  # '</ogc:PropertyIsGreaterThan>',
-  # ...
-  # '</ogc:And>',
   filterXML <- paste0('<?xml version="1.0"?>',
                       '<wfs:GetFeature xmlns:wfs="http://www.opengis.net/wfs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:gml="http://www.opengis.net/gml" service="WFS" version="1.1.0" outputFormat="application/json" xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">',
                       '<wfs:Query xmlns:feature="http://gov.usgs.cida/nhdplus" typeName="feature:nhdflowline_network" srsName="EPSG:4326">',
                       '<ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">',
+                      '<ogc:And>',
+                      '<ogc:PropertyIsGreaterThan>',
+                      '<ogc:PropertyName>streamorde</ogc:PropertyName>',
```

Comment:
should `'<ogc:PropertyName>streamorde</ogc:PropertyName>'` be `'<ogc:PropertyName>streamorder</ogc:PropertyName>'` ? missing the last `r` in `streamorder`